woocommerce/woocommerce
# [CYS on Core] Update the themes list on the intro screen

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

1. Ensure the following themes are listed on the Intro Screen for the implementation on core:

* [TT4](https://woo.local/wp-admin/themes.php?theme=twentytwentyfour)
* [Highline](https://woo.com/products/highline/)
* [Luminate](https://woo.com/products/luminate/)
* [Nokul](https://woo.com/products/nokul/)

2. Add the "Paid" badge
3. Add the color palette for the TT4 theme. The color palettes for the other themes will be added as part of a separate PR.

PS: there's also a pending change regarding a copy after the user clicks on each one of these themes, this change is planned to be implemented on a separate PR.

## Views

### Core (with one of the themes active)

<img width="1594" alt="Screenshot 2024-02-21 at 17 57 48" src="https://github.com/woocommerce/woocommerce/assets/15730971/99d8c0f1-dc48-407d-a20a-4d0d88aacfcc">

### Core (with none of those themes active)

<img width="1606" alt="Screenshot 2024-02-21 at 17 57 17" src="https://github.com/woocommerce/woocommerce/assets/15730971/7d2d2e6f-a974-4944-918a-ad9c8a4e7719">

### WooExpress (unchanged)

<img width="1593" alt="Screenshot 2024-02-21 at 18 11 32" src="https://github.com/woocommerce/woocommerce/assets/15730971/5929712f-eb8a-4561-b379-1457c79b544e">





Closes #44609



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the WooCommerce Beta Tester plugin if you don't have it enabled yet
2. On your Dashboard, head over to Tools > WCA Test Helper > Features and enable the `customize-store`
3. Access your Dashboard and head over to WooCommerce > Home
5. Click on the "Customize Your Store" button
6. Make sure you see the following themes listed:

<img width="1594" alt="Screenshot 2024-02-21 at 17 57 48" src="https://github.com/woocommerce/woocommerce/assets/15730971/99d8c0f1-dc48-407d-a20a-4d0d88aacfcc">

7. Click on each one of the themes and make sure the links are all correct.
8. Repeat steps 3 to 5 on WooExpress, and at that end, make sure the following themes are listed:

<img width="1593" alt="Screenshot 2024-02-21 at 18 11 32" src="https://github.com/woocommerce/woocommerce/assets/15730971/5929712f-eb8a-4561-b379-1457c79b544e">



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Update the themes list on the Customize Your Store intro screen.

#### Comment 

</details>
